src/main/groovy/com/github/jengelman/gradle/plugins/shadow/ShadowBasePlugin.groovy

```diff
@@ -13,8 +13,8 @@ class ShadowBasePlugin implements Plugin<Project> {
 
     @Override
     void apply(Project project) {
-        if (GradleVersion.current() < GradleVersion.version("8.0")) {
-            throw new GradleException("This version of Shadow supports Gradle 8.0+ only. Please upgrade.")
+        if (GradleVersion.current() < GradleVersion.version("8.3")) {
+            throw new GradleException("This version of Shadow supports Gradle 8.3+ only. Please upgrade.")
```

Comment:
Bump the min Gradle to 8.3, cause the `FilePermissions` has been added from 8.3, see https://docs.gradle.org/8.3/javadoc/org/gradle/api/file/FilePermissions.html

Reply:
Fine by me